demo/vl-map-draw-actions.html

```diff
@@ -49,6 +48,35 @@
                 </vl-map-features-layer>
             </vl-map>
         </vl-demo>
+
+        <vl-demo data-vl-title="Kaart met punt teken actie met snapping">
+            <vl-map id="map-with-draw-point-snap-action">
+                <vl-map-search></vl-map-search>
+                <vl-map-baselayer-grb-gray></vl-map-baselayer-grb-gray>
+                <vl-map-features-layer id="point-layer">
+                    <vl-map-layer-circle-style></vl-map-layer-circle-style>
+                    <vl-map-draw-point-action id="draw-point-snap-action" data-vl-default-active data-vl-snapping data-vl-snapping-pixel-tolerance="1000">
+                         <vl-map-wfs-layer data-vl-name="Stromend waterlichamen"
```

Comment:
Zou het niet beter zijn om de layers configuratie rechtstreeks onder de kaart te laten en bij de layers ook een optie data-vl-snapping te geven, zodat men weet waarop gesnapt moet worden?

Reply:
Dat heeft als nadeel dat die lagen dan altijd worden toegevoegd op de kaart, of dat ge daar weer met een attribuut moet zitten werken om aan te geven dat dat snapping lagen zijn ...

Reply:
Nu voeg ik enkel die laag toe aan de kaart als de actie actief wordt en haal ik ze er terug af bij deactivatie.

Reply:
Dus voorlopig OK me dunkt.